sound/soc/sof/intel/hda-dsp.c

```diff
@@ -467,26 +467,35 @@ static int hda_dsp_set_D0_state(struct snd_sof_dev *sdev,
 
 		if (hda_dsp_d0i3_playback_active(sdev))
 			flags |= HDA_PM_PG_STREAMING;
+
```

Comment:
simplify commit message with shorter sentences and bullet points.

The correct order should be : 
a) send ipc first 
b) set d0i3 register for enabling d0i3 
c) set d0i3 register to notify system power controller to change
power supply of audio hardware 
d) send ipc msg for disabling path.

Reply:
sure, thanks